More Highsierra work

jacobrosenthal opened the pull request:
ping #3

deadprogram commented on darwin/device.go:
```diff
@@ -516,7 +517,7 @@ func (d *Device) HandleXpcEvent(event xpc.Dict, err error) {
 			break
-		}
+		// }
 		c.rspc <- m
```
In other words, this line can, and should be removed, as it is no longer needed.

jacobrosenthal replied:
@moogle19 done, but then that break means the go channel thing on the next line is never happening..

deadprogram replied:
The line `c.rspc <- m` is to send message m to the channel c.rspc if not handled by the preceding code. Since your change handled m, no need to send now. That is why go get says that line should be removed.

deadprogram commented:
@jacobrosenthal pretty please with sugar on top!

jacobrosenthal commented:
Rebased to pull that commented code out. I would test this though, Im not a go guy and I barely understand that code flow. Also fixed a few more xpcids from reports on the noble side.

deadprogram commented:
@moogle19 seems like this is ready to merge, no?

jacobrosenthal commented:
So I've found via noble the args.isNotification wasn't a goble implementation anomaly, but was actually removed from the xpcids. So I'm now confident in that removal. Weve shipped these xpcids in noble 1.8.0 as of today.

moogle19 commented:
@jacobrosenthal Thanks a lot!
